quantum/audio/song_list.h

```diff
@@ -256,299 +235,15 @@
     E__NOTE(_E6),     \
     S__NOTE(_B5),
 
-#define COIN_SOUND \
-    E__NOTE(_A5  ),      \
-    HD_NOTE(_E6  ),
```

Comment:
I was going to argue that this would probably be ok to keep, but then [I found this](http://www.nintendolife.com/news/2016/03/nintendo_seeks_a_trademark_for_mario_coin_sound) :/

Reply:
Yeah.  I know that nintendo can be super vindictive about copyright stuff. It's why I pulled it.  Taking no risk with them.